roles/st2/handlers/main.yml

```diff
@@ -1,6 +1,7 @@
 - name: restart st2
   sudo: true
-  service:
-    name: "{{ item }}"
-    state: restarted
-  with_items: st2_services
+  command: st2ctl restart 
+
+- name: restart st2api
```

Comment:
We need to also restart st2stream along with st2api for pretty much every case. See https://github.com/StackStorm/st2-packages/pull/293/files. If you don't do that, attempt to connect to stream with token will result in an error, something along the lines 'unexpected query parameter x-auth-token'.